### PR TITLE
Payment

### PR DESCRIPTION
**Pull request by n-s-mordvinov**

### CODE REVIEW

File: src/components/payment/card/index.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    event.preventDefault();
+
+    if (!stripe || !elements) {
+      return;
+    }
+    const payload = await stripe.createPaymentMethod({
+      type: "card",
+      card: elements.getElement(CardElement)
+    });
+    console.log("[PaymentMethod]", payload);
```
Comment (kirillta) [inline src/components/payment/card/index.jsx:26]: Make at least a TODO to remove that line later

File: src/components/payment/index.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    text: `${data.firstName} ${data.lastName}`
+  })
+
+  const onChangeCheckedServiceFee = (e) => {
+    setCheckedServiceFee(!checkedServiceFee);
+  }
+
+  const onChangeAmount = (value) => {
+    let result = amount;
+    if (/^[\d]*\.?[\d]{0,2}$/.test(value)) {
```
Comment (kirillta) [inline src/components/payment/index.jsx:38]: Delimiter are different from culture to culture. Hardcode `.` isn't a good idea

File: src/components/payment/index.jsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      result = value;
+    }
+    setAmount(result);
+  }
+
+  const totalAmount = () => {
+    if (checkedServiceFee && parseFloat(amount)) {
+      return parseFloat(amount) + payment.proFormaInvoice.serviceFee.amount;
+    } else if (checkedServiceFee && !parseFloat(amount)) {
+      return 0 + payment.proFormaInvoice.serviceFee.amount;
```
Comment (kirillta) [inline src/components/payment/index.jsx:48]: `0 + ` looks redundant

File: src/components/ui/Switch/index.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      onClick={ () => switchRef.current.checked }
+    >
+      <input
+        type="checkbox"
+        ref={ switchRef }
+        className={ cn(style.switchInput, className) }
+        onChange={ onChange }
+        checked={ checked }
+      />
+      <span className={ style.switchChecked  }></span>
```
Comment (kirillta) [inline src/components/ui/Switch/index.jsx:21]: Double space here. Don't you use any linters?

File: src/components/payment/index.jsx (hunk 3 above)
Comment (sernik73) [reply src/components/payment/index.jsx]: Done

File: src/components/payment/card/index.jsx (hunk 1 above)
Comment (sernik73) [reply src/components/payment/card/index.jsx]: Added display for results

File: src/components/ui/Switch/index.jsx (hunk 4 above)
Comment (sernik73) [reply src/components/ui/Switch/index.jsx]: Done

**Review by kirillta — APPROVED** [review]

**Pull request merged**